[2023-11-30T03:12:24.629+0200] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: lviv_weather.inject_data_Kyiv manual__2023-11-30T01:12:14.705759+00:00 [queued]>
[2023-11-30T03:12:24.637+0200] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: lviv_weather.inject_data_Kyiv manual__2023-11-30T01:12:14.705759+00:00 [queued]>
[2023-11-30T03:12:24.638+0200] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-11-30T03:12:24.650+0200] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): inject_data_Kyiv> on 2023-11-30 01:12:14.705759+00:00
[2023-11-30T03:12:24.652+0200] {standard_task_runner.py:57} INFO - Started process 170485 to run task
[2023-11-30T03:12:24.656+0200] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'lviv_weather', 'inject_data_Kyiv', 'manual__2023-11-30T01:12:14.705759+00:00', '--job-id', '209', '--raw', '--subdir', 'DAGS_FOLDER/lviv_weather.py', '--cfg-path', '/tmp/tmphpeoxtpe']
[2023-11-30T03:12:24.658+0200] {standard_task_runner.py:85} INFO - Job 209: Subtask inject_data_Kyiv
[2023-11-30T03:12:24.716+0200] {task_command.py:416} INFO - Running <TaskInstance: lviv_weather.inject_data_Kyiv manual__2023-11-30T01:12:14.705759+00:00 [running]> on host yurabd.
[2023-11-30T03:12:25.023+0200] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='lviv_weather' AIRFLOW_CTX_TASK_ID='inject_data_Kyiv' AIRFLOW_CTX_EXECUTION_DATE='2023-11-30T01:12:14.705759+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-11-30T01:12:14.705759+00:00'
[2023-11-30T03:12:25.023+0200] {sql.py:274} INFO - Executing: 
            INSERT INTO measures (city, timestamp, date, temp, humidity, clouds, wind_speed) VALUES
            ('Kyiv',
            to_timestamp(1701306734),
            '2023-11-30 01:12:14',
            -4.23,
            81,
            82,
            0.89);
            
[2023-11-30T03:12:25.034+0200] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-11-30T03:12:25.063+0200] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-11-30T03:12:25.071+0200] {sql.py:418} INFO - Running statement: 
            INSERT INTO measures (city, timestamp, date, temp, humidity, clouds, wind_speed) VALUES
            ('Kyiv',
            to_timestamp(1701306734),
            '2023-11-30 01:12:14',
            -4.23,
            81,
            82,
            0.89);
            , parameters: None
[2023-11-30T03:12:25.073+0200] {sql.py:427} INFO - Rows affected: 1
[2023-11-30T03:12:25.089+0200] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=lviv_weather, task_id=inject_data_Kyiv, execution_date=20231130T011214, start_date=20231130T011224, end_date=20231130T011225
[2023-11-30T03:12:25.110+0200] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-30T03:12:25.135+0200] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
